code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -12,6 +12,13 @@
 	select_fire(user)
 	update_icon()
 
+/obj/item/weapon/gun/energy/gun/hos
+	desc = "This is a modern recreation of the antique laser gun. This gun has several unique firemodes, but lacks the ability to recharge over time, its also expensive."
+	icon_state = "hoslaser"
+	item_state = null
+	force = 10
+	ammo_type = list(/obj/item/ammo_casing/energy/electrode/hos, /obj/item/ammo_casing/energy/laser/hos, /obj/item/ammo_casing/energy/laser/pulse/breach)
+

```

Comment:
The gun should have it's own name set, or it'll be confusing when fired and a pain in the logs